packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -750,3 +752,173 @@ function CustomTab(props) {
       })} />
   );
 }
+
+function Draggable() {
+  let buttonRef = useRef(null);
+  let {dragProps, isDragging} = useDrag({
+    getItems() {
+      return [{
+        'text/plain': 'hello world'}];
+    }
+  });
+  let {buttonProps} = useButton({elementType: 'div'}, buttonRef);
+
+  return (
+    <FocusRing focusRingClass={classNames(styles, 'focus-ring')}>
+      <div
+        {...mergeProps(buttonProps, dragProps)}
+        ref={buttonRef}
+        className={classNames(styles, 'draggable', {['dragging']: isDragging})}>
+        Drag me
+      </div>
+    </FocusRing>
+  );
+}
+
+function Copyable() {
+  let {clipboardProps} = useClipboard({
+    getItems() {
+      return [{
+        'text/plain': 'hello world'
+      }];
+    }
+  });
+
+  return (
+    <FocusRing focusRingClass={classNames(styles, 'focus-ring')}>
+      <div 
+        {...clipboardProps}
+        role="textbox" 
+        tabIndex={0}
+        className={styles.copyable}>
+        Copy me 
+      </div>
+    </FocusRing>
+  );
+}
+
+export const DropzoneExampleWithInput = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onPress={action('OnPress')}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <Text slot="heading">
+        DropZone Area
+      </Text>
+      <Input />
+    </DropZone>
+  </div>
+);
+
+export const DropzoneExampleWithLinkAndInput = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onPress={action('OnPress')}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <Input />
+      <Link slot="file">Upload</Link>
+    </DropZone>
+  </div>
+);
+
+export const DropzoneExampleWithButton = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onPress={action('OnPress')}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <Button slot="file">upload</Button>
+      <Input />
+    </DropZone>
+  </div>
+);
+
+export const DropzoneExampleWithDraggableObject = (props) => (
+  <div>
+    <Draggable />
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onPress={action('OnPress')}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')} >
+      <Text slot="heading">
+        DropZone Area
+      </Text>
+    </DropZone>
+  </div>
+);
+
+export const DropzoneExampleWithCopyableObject = (props) => (
+  <div>
+    <Copyable />
+    <DropZone
```

Comment:
Is the VO for this complete? When I test Chrome on OSX it seems like it's not announcing some drop events.

Reply:
the VO should be very similar to dnd so do you know what drop events that it wasn't annoucning?